### DISCUSSION TITLE
Unable to send keystrokes to the game.

### DISCUSSION THREAD

**Discussion by Reyzirk** [tanjeffreyz/auto-maple#144] [Q&A]
I've tried multiple times in an attempt to start up the macro while the game is running. I've set up test routines and keystrokes (I modified the adele.py resources file to my corresponding entries) does not register to the game. Is there any solution to this issue? I've read discussions and it seems auto-maple needs "elevated privileges" based on an answer, but how "elevated privileges" are assigned to an entire folder/Python file?

**Comment by ziya000** [comment]
i had the same issue, you need to run the code on admin level to send keystrokes to the game.

not sure how you are running the code. what i did is i directly modified and run main.py (i did not use setup.py). you can modify main.py based on this post https://stackoverflow.com/questions/130763/request-uac-elevation-from-within-a-python-script/41930586#41930586.

put the code in main function under the `if is_admin():` statement
